storage/rocksdb/CMakeLists.txt

```diff
@@ -82,14 +101,10 @@ ENDIF()
 
 SET(rocksdb_static_libs ${rocksdb_static_libs} ${ZLIB_LIBRARY} "-lrt")
 
-MYSQL_ADD_PLUGIN(rocksdb_se ${ROCKSDB_SOURCES} STORAGE_ENGINE DEFAULT STATIC_ONLY
+MYSQL_ADD_PLUGIN(rocksdb ${ROCKSDB_SOURCES} STORAGE_ENGINE DEFAULT MODULE_ONLY
   LINK_LIBRARIES ${rocksdb_static_libs}
 )
 
-IF(WITH_EMBEDDED_SERVER)
```

Comment:
Trying to build an embedded PS actually works in 5.6 and 5.7 - do you always build with -DWITH_EMBEDDED_SERVER=OFF? If yes, then just remove that option.